templates/modelExpt/gen_fmtMatrix_modelComp.m

```diff
@@ -0,0 +1,43 @@
+function [savefile] = gen_fmtMatrix_modelComp(dataPath,dataValsStr,bSaveCheck)
+% A wrapper around gen_fmtMatrixByCond designed to be used with data from
+% run_modelComp_expt. See gen_fmtMatrixByCond for more info.
+
+if nargin < 1 || isempty(dataPath), dataPath = cd; end
+if nargin < 2 || isempty(dataValsStr), dataValsStr = 'dataVals_audapter.mat'; end
+if nargin < 3 || isempty(bSaveCheck), bSaveCheck = 1; end
+
+load(fullfile(dataPath,'expt.mat'),'expt');
+
+%add shift information to expt.mat
+if ~isfield(expt,'shifts')
+    expt.shifts.mels{1} = [125 0];
+    expt.shifts.mels{2} = [-125 0];
+end
+save(fullfile(dataPath,'expt.mat'),'expt');
```

Comment:
Make sure that expt.shifts.mels is set in run_modelComp_expt, and then remove this -- I believe it was there for an older experiment where we forgot to save this variable when collecting data.